core-primitives/node-api/api-client-extensions/src/pallet_teerex_api_mock.rs

```diff
@@ -30,7 +31,7 @@ impl PalletTeerexApiMock {
 	}
 }
 
-impl PalletTeerexApi for PalletTeerexApiMock {
+impl PalletTeerexApi<Runtime> for PalletTeerexApiMock {
```

Comment:
If you do the above suggested change, we can also remove the `Runtime` here and manually set the hash to `H256` here.